project/Build.scala

```diff
@@ -1176,6 +1193,17 @@ object Build extends sbt.Build {
     compiler % "plugin", library, jUnitRuntime % "test", jasmineTestFramework % "test"
   )
 
+  lazy val testSuiteJVM = Project(
+    id = "testSuiteJVM",
+    base = file("test-suite/shared"),
```

Comment:
Makes sense. I was just imitating the structure of `CrossProject`s which ends up being confusing.